### ISSUE TITLE
Missing parenthesis in loss

### ISSUE THREAD

**Issue by ctiennot** [MasazI/cnn_depth_tensorflow#3]
Hi again, 

In the file model.py at line 46, shouldn't:
`cost = tf.reduce_mean(sum_square_d / 55.0*74.0 - 0.5*sqare_sum_d / math.pow(55*74, 2))`

be replaced by 
`cost = tf.reduce_mean(sum_square_d / (55.0*74.0) - 0.5*sqare_sum_d / math.pow(55*74, 2))` 

to match equation (4) form the paper? (I think you just forgot the parenthesis).

Best, 

Clement

**Comment by drstarson** [comment]
@ctiennot Did you succeed training the model and getting the results after adding the parenthesis. It looks like the parenthesis are needed, but after adding them, the system doesn't seem to be training well. I'm getting low loss (~0.2), but the resulting depth images are all black. Would appreciate any advice or additional info.

**Comment by josslynZcn** [comment]
@drstarson  i have experience the same problem with you. and i wonder if you solve this problem .would  appreciate with the solution about this problen. i would be so thankful if you connect with me zhangchenni121@126.com
